Makefile

```diff
@@ -17,12 +25,34 @@ node-deps: node_modules/
 node_modules/: package.json
 	npm install
 
+local-components: node-deps
+	cd $(shell dirname $(current_dir))
+	if [ ! -e "$(shell dirname $(shell pwd))/lev-react-components" ] ; then \
+		echo 'ERROR: Please clone the necessary lev-react-components project in the parent directory' ; \
+		exit 1 ; \
+	fi
+	rm -fr node_modules/lev-react-components/dist
+	ln -s $(shell dirname $(shell pwd))/lev-react-components/dist node_modules/lev-react-components/dist
```

Comment:
Nice. It's good to have this codified. In the long run, of course. we should have a better way of working on the components in isolation. (Probably with Storybook.)